content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html

```diff
@@ -0,0 +1,49 @@
+---
+title: Tutorial interativo - implantando um aplicativo
+weight: 20
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+<link href="/docs/tutorials/kubernetes-basics/public/css/overrides.css" rel="stylesheet">
```

Comment:
Should these be links to `/pt/*` ?

Reply:
I can see all other localizations are not specifying the locale on the link reference

i.e [here](https://github.com/kubernetes/website/blob/master/content/fr/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html#L12) and [here](https://github.com/kubernetes/website/blob/master/content/es/docs/tutorials/kubernetes-basics/deploy-app/deploy-interactive.html#L12)

Reply:
do you think we should start localizing those even if are duplicated and with the risk of stale it?